src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1280,7 +1282,12 @@ private void buildContainerImage(File tempContainerfile, File userContainerfile,
                 info("The RUN features.sh command is detected in the Containerfile and extra time may be necessary when installing features.");
             }
             long startTime = System.currentTimeMillis();
-            execContainerCmdAndLog(getRunProcess(buildCmd), containerBuildTimeout);
+            //Run build command with an Ant task if using Podman on Windows, see https://github.com/OpenLiberty/ci.maven/issues/1746
+            if (!isDocker && OSUtil.isWindows()) {
+                containerAntRun(buildCmd);
+            } else {
+                execContainerCmdAndLog(getRunProcess(buildCmd), containerBuildTimeout);
```

Comment:
There are some things that this method does we need to account for in this new `containerAntRun` method. For example, the `writeDevcMetadata(true)` is called. There is also some logic around stopping dev mode if the container does not start successfully. And there are some threads to capturing the input and error stream.